exercises/bob/description.md

```diff
@@ -4,6 +4,8 @@ Bob answers 'Sure.' if you ask him a question.
 
 He answers 'Whoa, chill out!' if you yell at him.
 
+He answers 'Calm down, I know what I'm doing!' if you question him emphaticlly.
```

Comment:
spelling issue on this line.

Also a bit worried about complex language use here, since there are many non-english speakers.

Reply:
Yup, I see the spelling issue now.  I chose `emphatically` because it helped to keep the line short too.  

...`if you yell your question at him.`  

should suffice.